recipes-wolfssl/wolftpm/files/0001-apply-git-PR-196.patch

```diff
@@ -0,0 +1,370 @@
+diff --git a/examples/pcr/include.am b/examples/pcr/include.am
```

Comment:
There is a wolfTPM v2.3.1 tag that can avoid this patch. Would it be easier to just use that? https://github.com/wolfSSL/wolfTPM/releases/tag/v2.3.1

Reply:
The recipes are currently set up to just download the release .zip's.  We could rework to clone via GitHub, but would probably want to rework for all recipes to be consistent.  If we want to go that route, I can work on adjusting.

Reply:
Actually, disregard, maybe we can just switch to downloading the .zip from GitHub, ie:
`https://github.com/wolfSSL/wolfTPM/archive/refs/tags/v2.3.1.zip`.  I'll work to switch this.

Reply:
Recipes have been updated to pull from a git tag, which removes the need for wolfTPM patch file when using tag `v2.3.1`.